# Publish all joint states ----------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

fullbody_controller:
  type: velocity_controllers/JointTrajectoryController
  joints:
    - front_left_shoulder
    - front_left_leg
    - front_left_foot
    - front_right_shoulder
    - front_right_leg
    - front_right_foot
    - rear_left_shoulder
    - rear_left_leg
    - rear_left_foot
    - rear_right_shoulder
    - rear_right_leg
    - rear_right_foot
  gains:
    front_left_shoulder: {p: 250, i: 0, d: 0}
    front_left_leg: {p: 250, i: 0, d: 0}
    front_left_foot: {p: 250, i: 0, d: 0}
    front_right_shoulder: {p: 250, i: 0, d: 0}
    front_right_leg: {p: 250, i: 0, d: 0}
    front_right_foot: {p: 250, i: 0, d: 0}
    rear_left_shoulder: {p: 250, i: 0, d: 0}
    rear_left_leg: {p: 250, i: 0, d: 0}
    rear_left_foot: {p: 250, i: 0, d: 0}
    rear_right_shoulder: {p: 250, i: 0, d: 0}
    rear_right_leg: {p: 250, i: 0, d: 0}
    rear_right_foot: {p: 250, i: 0, d: 0}
